sys/netbsd/lwp.txt

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <sys/types.h>
+include <sys/lwp.h>
+include <sys/lwpctl.h>
+include <sys/ucontext.h>
+include <sys/signal.h>
+
+resource lwpid[int32]: 0, 0xffffffffffffffff
+
+_lwp_create(context ptr[in, ucontext_t], cflags flags[lwp_flags], lwpid ptr[out, lwpid])
+_lwp_exit()
+_lwp_self(lwp lwpid)
+_lwp_wait(wlwp lwpid, rlwp ptr[in, lwpid])
+_lwp_suspend(lwp lwpid)
+_lwp_continue(lwp lwpid)
+_lwp_wakeup(lwp lwpid)
+_lwp_getprivate()
+_lwp_setprivate(data ptr[in, intptr])
+_lwp_kill(lwp lwpid, sig int32)
+_lwp_detach(lwp lwpid)
+compat_50__lwp_park(abstime ptr[in, timespec], unpark lwpid, hint ptr[in, intptr, opt], unparkhint ptr[in, intptr, opt])
+_lwp_unpark(lwp lwpid, hint ptr[in, intptr, opt])
+_lwp_unpark_all(lwp lwpid, size intptr, hint ptr[in, intptr, opt])
```

Comment:
`s/lwp lwpid, size intptr/targets ptr[in, array[lwpid]], ntargets len[targets]/`

Please stick to the argument names used in the kernel sources. It will be useful if/when we have some kind of static analysis/verification.
Though I only looked at the man:
https://netbsd.gw.com/cgi-bin/man-cgi?_lwp_unpark_all+2+NetBSD-current
Are they called target/ntargets in kernel too?